l10n/po/zh_CN/_guides/kafka.adoc.po

```diff
@@ -129,45 +123,39 @@ msgstr ""
 
 #. type: delimited block =
 #: upstream/_guides/kafka.adoc:65
-#, fuzzy
 msgid "The extension includes `kafka-clients` version 3.1.0 as a transitive dependency and is compatible with Kafka brokers version 2.x."
-msgstr "该扩展包括 `kafka-clients` 3.1.0版本作为一个横向依赖，并与Kafka经纪人2.x版本兼容。"
+msgstr "该扩展包括 `kafka-clients` 3.1.0版本的传递依赖，而且与Kafka broker 2.x版本兼容。"
 
 #. type: Title ==
 #: upstream/_guides/kafka.adoc:67
-#, fuzzy, no-wrap
+#, no-wrap
 msgid "Configuring Smallrye Kafka Connector"
-msgstr "配置Smallrye Kafka连接器"
+msgstr "配置Smallrye Kafka Connector"
 
 #. type: Plain text
 #: upstream/_guides/kafka.adoc:70
-#, fuzzy
 msgid "Because Smallrye Reactive Messaging framework supports different messaging backends like Apache Kafka, AMQP, Apache Camel, JMS, MQTT, etc., it employs a generic vocabulary:"
-msgstr "因为Smallrye Reactive Messaging框架支持不同的消息传递后端，如Apache Kafka、AMQP、Apache Camel、JMS、MQTT等，所以它采用了一个通用词汇表。"
+msgstr "因为Smallrye Reactive Messaging框架支持不同的消息后端，如Apache Kafka，AMQP，Apache Camel，JMS以及MQTT等，所以它使用了一个通用的术语表："
 
 #. type: Plain text
 #: upstream/_guides/kafka.adoc:72
-#, fuzzy
 msgid "Applications send and receive *messages*. A message wraps a _payload_ and can be extended with some _metadata_. With the Kafka connector, a _message_ corresponds to a Kafka _record_."
-msgstr "应用程序发送和接收 *消息* 。一条消息包裹着一个 _有效载荷_ ，并可以用一些 _元数据_ 进行扩展。通过Kafka连接器，一条 _消息_ 对应于一条Kafka _记录_ 。"
+msgstr "应用程序发送和接收 *messages* 。一条消息包含一个 _payload_ ，并可以用一些 _metadata_ 进行扩展。通过Kafka connector，一条 _message_ 对应于一条Kafka_record_。"
 
 #. type: Plain text
 #: upstream/_guides/kafka.adoc:73
-#, fuzzy
 msgid "Messages transit on *channels*. Application components connect to channels to publish and consume messages. The Kafka connector maps _channels_ to Kafka _topics_."
-msgstr "信息在 *通道* 上传输。应用程序组件连接到通道来发布和消费消息。Kafka连接器将 _通道_ 映射到Kafka _主题_ 。"
+msgstr "信息在 *通道* 上传输。应用程序组件通过连接通道来发布和消费消息。Kafka connector将 _通道_ 映射到Kafka的 _主题_上 。"
```

Comment:
It could be better to leave with `channels` and `topics` and also pay attension to the white spaces.